### PR TITLE
May / June Grant Updates

### PR DESCRIPTION
**Pull request by jakep**
**Description**
- Updated the grant total
- Updated grantee history
- Add June focus area
- Add June grantee

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -24,13 +24,13 @@
 # twitter_username: jekyllrb
 # github_username:  jekyll
 
-contributions_to_date: $25,626
+contributions_to_date: $30,948
 
 current_focus:
-  title: Immigration
+  title: Environment Justice
```
Comment (nuprin) [inline _config.yml]: `Environmental Justice`

File: _config.yml [hunk 2]
```diff
@@ -24,13 +24,13 @@
...
   # Note: have to cap off the body with "…" manually:
-  body: "The results are in! With 47% of all votes, Immigration has been selected by the community as our May focus. Our advisory committee will now begin researching and vetting nonprofits that meet our criteria: Fighting for equity and justice in America: For immigration, this could mean an organization providing legal assistance, workforce training, or education programs."
-  url: https://blog.vocalminority.us/our-may-focus-immigration-5c447e14600a
+  body: "Given the continued rhetoric around climate change and the recent news of the US pulling out of The Paris Agreement, our community has decided to again focus on environmental justice. Over 50% of our members voted for this focus, which is evident of the severity and importance of this issue."
```
Comment (nuprin) [inline _config.yml]: Suggested rewording

> “With the United States announcing its withdrawal from the Paris Climate Agreement, our community has decided to again to focus on environmental justice. Over 50% of our members voted for this focus, demonstrating how critical we feel this issue is in the fight for a more equitable and just world.“

File: _config.yml [hunk 3]
```diff
@@ -40,20 +40,31 @@ current_focus:
...
-  url: https://blog.vocalminority.us/vocal-minority-funds-legal-rights-for-immigrants-394929ad7ebc
+  body: "Alliance for Climate Education’s mission is to educate young people on the science of climate change and empower them to take action."
+  url: https://acespace.org/
 
 grantees:
+  - month: May
+    year: 2017
+    name: "Las Americas Immigrant Advocacy Center"
+    short_name: Las Americas # On home page
+    description: Las Americas is a 29 year-old, homegrown, non-profit dedicated to serving the legal needs of low income immigrants, including refugees, victims of crime, and families seeking reunification.
```
Comment (nuprin) [inline _config.yml]: > "29-year-old"

File: _config.yml (hunk 1 above)
Comment (jakep) [inline _config.yml]: Good catch 👍

**Pull request merged**